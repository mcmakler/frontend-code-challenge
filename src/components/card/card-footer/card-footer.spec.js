import React from 'react';
import CardFooter from './card-footer';
import ReactTestRenderer from 'react-test-renderer';

describe('CardFooter specs', () => {
  it('should match snapshot', () => {
    const props = {
      price: {
        value: '',
        label: ''
      },
      rooms: {
        value: '',
        label: ''
      },
      space: {
        value: '',
        label: ''
      }
    };
    const component = ReactTestRenderer.create(<CardFooter {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
