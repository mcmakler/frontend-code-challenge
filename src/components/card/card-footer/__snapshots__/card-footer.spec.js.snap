// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardFooter specs should match snapshot 1`] = `
<div
  className="cardFooter"
>
  <div
    className="cardFooter__item cardFooter__item--bold"
  >
    
     
    
  </div>
  <div
    className="cardFooter__item cardFooter__item--separated cardFooter__item--light"
  >
    <span>
      
       
      
    </span>
    <span>
      
       
      
    </span>
  </div>
</div>
`;
